tools/cosmovisor/cmd/cosmovisor/init.go

```diff
@@ -95,6 +100,15 @@ func getConfigForInitCmd() (*cosmovisor.Config, error) {
 		Home: os.Getenv(cosmovisor.EnvHome),
 		Name: os.Getenv(cosmovisor.EnvName),
 	}
+
+	var err error
+	if cfg.ColorLogs, err = cosmovisor.BooleanOption(cosmovisor.EnvColorLogs, true); err != nil {
+		errs = append(errs, err)
+	}
+	if cfg.TimeFormatLogs, err = cosmovisor.TimeFormatOptionFromEnv(cosmovisor.EnvTimeFormatLogs, time.Kitchen); err != nil {
+		errs = append(errs, err)
+	}
+
```

Comment:
L22 should return the config.

Reply:
didn't get it, another file?

Reply:
in `getConfigForInitCmd`, when there an err we still want to return the config, as we discard the error on main.
Otherwise it panics with a nil pointer (when we access the logger)

Reply:
do you refer to L122?

Reply:
Yeah